about.html

```diff
@@ -0,0 +1,190 @@
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <title>About Page - ReviewMe</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navbar fixed-top navbar-expand-md navbar-light bg-light">
+        <div class="container-fluid">
+          <a class="navbar-brand nav-bar-logo-image" href="index.html">
+            <img
+              src="assets/images/reviewme-website-favicon-color-updated.png"
+              alt="logo image"
+            />
+          </a>
+          <button
+            class="navbar-toggler collapsed d-flex d-md-none flex-column justify-content-around"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarNav"
+            aria-controls="navbarNav"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="toggler-icon top-bar"></span>
+            <span class="toggler-icon middle-bar"></span>
+            <span class="toggler-icon bottom-bar"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+              <li class="nav-item me-5 home-menu-about">
+                <a class="nav-link" aria-current="page" href="index.html"
+                  >Home</a
+                >
+              </li>
+              <li class="nav-item me-5">
+                <a
+                  class="nav-link about-active"
+                  aria-current="page"
+                  href="index.html"
+                  >About</a
+                >
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link" href="index.html#service-section"
+                  >Services</a
+                >
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link" href="index.html#partners-section"
+                  >Sponsors</a
+                >
+              </li>
+              <li class="nav-item me-5">
+                <a class="nav-link">Forum</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="banner-section banner-section-about-page">
+        <div class="background-image background-image-about-page"></div>
+        <div class="banner-text-content banner-text-content-about">
+          <h1 class="mb-3 text-center">Hello Developers!</h1>
+          <h2 class="fw-bold text-center">
+            ReviewMe, GET INSTANT HELP, <br />
+            MEET PROFESSIONALS AND <br />
+            MOVE 5x FASTER
+          </h2>
```

Comment:
- [ ] This is the main title of your page, so it has to be written using the `h1` tag.